packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -3538,8 +3549,17 @@ class _DropSliderValueIndicatorPathPainter {
       ..lineTo(-_triangleHeight, -_triangleHeight)
       ..lineTo(_triangleHeight, -_triangleHeight)
       ..close();
+    trianglePath.addRRect(borderRect);
+
+    if (strokePaintColor != null) {
+      final Paint strokePaint = Paint()
+        ..color = strokePaintColor
+        ..strokeWidth = 1.0
+        ..style = PaintingStyle.stroke;
+      canvas.drawPath(trianglePath, strokePaint);
+    }
+
     canvas.drawPath(trianglePath, fillPaint);
-    canvas.drawRRect(borderRect, fillPaint);
 
```

Comment:
@TahaTesser 
Are these modifications accurate for incorporating stroke colors into the DropSliderValueIndicatorShape?

Reply:
Looks good